SeQuant/domain/eval/eval.hpp

```diff
@@ -535,6 +537,36 @@ ResultPtr evaluate_antisymm(Args&&... args) {
   }
   return result;
 }
+
+///
+/// \brief Calls @code evaluate followed by the biorthogonal-cleanup function.
```

Comment:
a couple of issues:
- each `@code` need a `@endcode` to close code block. To highlight a single word use back quotes
- this needs a more details dox. Ideally you can show the expression for the triples or refer to an equation in the paper (make sure to give DOI if referring to the paper)

to make sure documentation looks good you need to generate documentation and check it out; CI does not ensure that documentation looks correct, only that it builds

Reply:
if the biorthogonal-cleanup is documented elsewhere then can just refer to that instead of giving details here. Ideally there is one place where the cleanup is described.